src/chrome/content/rules/Frugalware.org.xml

```diff
@@ -0,0 +1,38 @@
+<!--
+	Invalid certficate:
+		darcs.frugalware.org
+		dors.frugalware.org
+		git.frugalware.org
+		lists.frugalware.org
+		packages.frugalware.org
+		planet.frugalware.org
+		wiki.frugalware.org
+
+	Private subdomain:
+		webmail.frugalware.org
+
```

Comment:
It looks like none of them are working over https, would you accept something like:

```xml
	Non functional over https for various reasons:
		www\d.frugalware.org
		rsync\d.frugalware.org
```

?